### ISSUE TITLE
about the open checkpionts accuracy did not inconsistent with yours.

### ISSUE THREAD

**Issue by 723373214** [wzekai99/DM-Improves-AT#2]
I have tried to evaluate your wrn-76-16 on cifar-10 under linf norm, but the clean accuracy is 73.71%, not 93% as you describe, how did it comes?

**Comment by wzekai** [comment]
Hi, 

Thank you for using our checkpoint.

Please test the model using the [environment setting](https://github.com/wzekai99/DM-Improves-AT#environment-settings-and-libraries-we-used-in-our-experiments) and [`eval-aa.py`](https://github.com/wzekai99/DM-Improves-AT/blob/main/eval-aa.py) we recommend. Our results are retested by [RobustBench](https://robustbench.github.io/). 

Or you can use the script below with the latest version of RobustBench:

```python
import torch
from robustbench.data import load_cifar10
from robustbench.utils import clean_accuracy, load_model

# Load a model from the model zoo
model = load_model(model_name='Wang2023Better_WRN-70-16', dataset='cifar10', threat_model='Linf')

device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
model = model.to(device)

clean_x_test, clean_y_test = load_cifar10()
accuracy = clean_accuracy(model, clean_x_test, clean_y_test, batch_size=512, device=device)
print(f'Clean accuracy: {accuracy:.2%}')
```
